components/support/nimbus-fml/src/backends/kotlin/templates/FeatureManifestTemplate.kt

```diff
@@ -59,18 +59,14 @@ object {{ nimbus_object }} {
     }
 
     private var getSdk: () -> FeaturesInterface? = {
-        this.api
+        null
     }
 
-    /**
-     * This should be populated at app launch.
-     *
-     * This property has to be set after the Nimbus SDK, which causes race-conditions. Please use
-     * `initialize()` much earlier in the application startup sequence.file
-     *
-     * This will be removed in future releases.
-     */
-    var api: FeaturesInterface? = null
```

Comment:
This will also be a breaking change right? (assuming Fenix still uses the old way of initialization, which I think it does since they didn't take our a-s release just yet)

Reply:
Also if we opt to remove this, should we also remove it from the swift manifest template?

Reply:
Yeah, I'll put this back.

Reply:
Put this back, for removal after the apps upgrade.

Reply:
did we put this back 😅 I'm not sure if it's a GitHub thing

Reply:
oops ya github didn't diplay the diff properly for me 😬